cluster/calcium/resource.go

```diff
@@ -85,8 +87,23 @@ func (c *Calcium) allocResource(ctx context.Context, opts *types.DeployOptions,
 			return nil, err
 		}
 	}
-	log.Debugf("[allocResource] result %v", nodesInfo)
-	return nodesInfo, nil
+	go func() {
+		log.Info("[allocResource] result")
+		for _, nodeInfo := range nodesInfo {
+			s := litter.Sdump(nodeInfo.CPUPlan)
+			log.Infof("[allocResource] deploy %d to %s \n%s", nodeInfo.Deploy, nodeInfo.Name, s)
+		}
+	}()
+	return nodesInfo, c.bindProcessStatus(ctx, opts, nodesInfo)
+}
+
+func (c *Calcium) bindProcessStatus(ctx context.Context, opts *types.DeployOptions, nodesInfo []types.NodeInfo) error {
+	for _, nodeInfo := range nodesInfo {
+		if err := c.store.SaveProcessing(ctx, opts, nodeInfo); err != nil {
```

Comment:
One fail, all fail? But the `doCreateContainer` simply `continue` to next one. Can be configurable here?

Reply:
This only affects the primary distribution and will be recalculated in next deploying. Fast fail will be better.